site/content/en/v1.0.1/user/quickstart.md

```diff
@@ -43,7 +43,7 @@ unprivileged port, so that Envoy Gateway doesn't need additional privileges.
 It's important to be aware of this mapping, since you may need to take it into
 consideration when debugging.
 
-[`quickstart.yaml`]: https://github.com/envoyproxy/gateway/releases/download/v1.0.0/quickstart.yaml
+[`quickstart.yaml`]: https://github.com/envoyproxy/gateway/releases/download/v1.0.1/quickstart.yaml
```

Comment:
Some links are 404 as expected since release is not cut off yet

Reply:
this link will never 404, because github.com is ignored because of rate limiting.